app/user/tests/test_user_api.py

```diff
@@ -0,0 +1,106 @@
+from django.test import TestCase
+from django.contrib.auth import get_user_model
+from django.urls import reverse
+
+from rest_framework.test import APIClient
+from rest_framework import status
+
+
+CREATE_USER_URL = reverse('user:create')
+TOKEN_URL = reverse('user:token')
+ME_URL = reverse('user:me')
+
+
+def create_user(**params):
+    return get_user_model().objects.create_user(**params)
```

Comment:
Out of the scope of this exercise, but have a look at https://factoryboy.readthedocs.io/en/latest/orms.html#django 
This allows you to create dummy entries for testing.

Reply:
ok, good to know :+1: